src/components/Onboarding.jsx

```diff
@@ -0,0 +1,126 @@
+import * as rb from 'react-bootstrap'
+import React, { useState } from 'react'
+import Sprite from './Sprite'
+
+export default function Onboarding() {
+  const [step, setStep] = useState(0)
+
+  if (step === 0) {
+    return (
+      <div class="mx-auto my-auto">
+        <div class="d-flex justify-content-center">
+          <Sprite symbol="logo" width="200" height="200" className="mb-4" />
+        </div>
+        <h1 class="text-center">JoinMarket</h1>
+        <p class="text-center fs-4 text-secondary mb-5">Top-notch privacy for your bitcoin.</p>
+        <div class="d-flex justify-content-center">
+          <rb.Button variant="dark" onClick={() => setStep(1)}>
+            Get started
+          </rb.Button>
+        </div>
+        <p class="mt-4 text-center">
+          Your wallet, your coins. <br />
+          100% open-source & open-design.
+        </p>
+      </div>
+    )
+  } else if (step === 1) {
+    return (
+      <div class="mx-auto my-auto col-12 col-md-4">
+        <div class="d-flex justify-content-center">
+          <Sprite symbol="welcome" width="100" height="100" className="mb-4" />
+        </div>
+        <p class="fs-3 text-center">1: Welcome to JoinMarket!</p>
+        <p class="text-center text-secondary mb-5">
+          JoinMarket is a privacy-focused software solution that aims to improve the confidentiality and privacy of your
+          bitcoin transactions. It facilitates the creation of collaborative transactions through a peer-to-peer
+          marketplace. <br />
+          <br />
+          ⚠️ Warning: While JoinMarket is tried and tested, this user interface is not. If anything breaks please{' '}
+          <a href="https://github.com/joinmarket-webui/joinmarket-webui/issues" target="_blank" rel="noreferrer">
+            report an issue
+          </a>{' '}
+          on GitHub.
+        </p>
+        <div class="d-flex justify-content-center">
+          <rb.Button variant="dark" onClick={() => setStep(2)}>
+            Continue
+          </rb.Button>
+        </div>
```

Comment:
Should we make the button a bit bigger maybe? Cf. the Figma screenshot below:

<img width="396" alt="Screenshot 2022-02-07 at 16 00 55" src="https://user-images.githubusercontent.com/10026790/152813225-0e8ed900-80b5-4ce6-9387-2e278dd35dcd.png">